entries/n/nicehash.com.json

```
@@ -0,0 +1,14 @@
+{
+  "NiceHash": {
+    "domain": "nicehash.com",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
```

```suggestion
    "tfa": [
      "totp",
      "custom-hardware"
    ],
    "custom-hardware": [
      "Yubico OTP"
    ],
```

The proprietary Yubico OTP system is being used rather than U2F. See https://www.nicehash.com/support/general-help/security/how-to-add-otp-device

---

Done. I was confused. So, u2f should be used when it supports all forms rather than just specific hardware like yubikey, right?

---

`u2f` should be used when a site supports U2F/WebAuthn, however this site only supports a proprietary method that is compatible with YubiKeys, called Yubico OTP, which is why the `custom-hardware` value was needed.